packages/oidc-dpop-client-browser/src/dpop/keyGen.ts

```diff
@@ -0,0 +1,53 @@
+/*
```

Comment:
There no need to abbreviate the filename here - I'd just call it `keyGeneration.ts`.

Reply:
Resolved in 44808e6e3ada27958ac7fa996102200c21f0f0ea, with a small fix in 77e1054e4d2abbb463c971b2bde0748739f212aa for the mock